test/parallel/test-assert-deep.js

```diff
@@ -1220,3 +1220,60 @@ assert.throws(
 
   assertNotDeepOrStrict(a, b);
 }
+
+// eslint-disable-next-line node-core/crypto-check
+if (common.hasCrypto) {
+  const crypto = require('crypto');
+  const { subtle } = globalThis.crypto;
+
+  {
+    const a = crypto.createSecretKey(Buffer.alloc(1, 0));
+    const b = crypto.createSecretKey(Buffer.alloc(1, 1));
```

Comment:
nit: I find `Buffer.from` easier to understand (I had to go read the docs to get what the second argument of `Buffer.alloc` was for)
```suggestion
    const a = crypto.createSecretKey(Buffer.from([0]));
    const b = crypto.createSecretKey(Buffer.from([1]));
```